packages/cover-traffic/src/state.ts

```diff
@@ -17,17 +19,20 @@ export type OpenChannels = {
 
 export type State = {
   nodes: Record<string, PeerData>
+  // channels indexed by its channelId
   channels: Record<string, ChannelData>
```

Comment:
out of scope of this PR: We could replace the `string` type with a custom `ChannelId` type.

Reply:
Noted

Reply:
worth noting that type aliases are essentially useless in typescript, and record _needs_ a stringlike key.